.github/workflows/ci.yml

```diff
@@ -0,0 +1,84 @@
+# https://help.github.com/en/categories/automating-your-workflow-with-github-actions
+
+on:
+  - "pull_request"
```

Comment:
I have started to consistently quote `string`s in YAML - if you prefer, I can remove the quotes, though.